### ISSUE TITLE
2021-12-6

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#676]
912019, cited by 165; 
92Abstract
931.Convolutional neural networks; exponential increment of the computational and storage cost; (2021-11-22)
942.self distillation, shrinking the size of the network rather than aggrandizing it; traditional knowledge distillation, forces student neural networks to approximate the softmax layer outputs of pre-trained teacher neural networks; self distillation framework distills knowledge within network itself; knowledge in the deeper portion of the networks is squeezed into the shallow ones; outperform; (2021-11-22)
95I.Introduction
961.shorter response time is required; Knowledge Distillation (KD) to achieve model compression; (2021-11-22)
972.Knowledge distillation [15] is inspired by knowledge transfer from teachers to students; knowledge distillation [15] is inspired by knowledge transfer from teachers to students; DRAWBACKS: (1) low efficiency on knowledge transfer; (2) how to design and train proper teacher models; (2021-11-22)
983.self distillation framework; propose a one-step self distillation framework whose training points directly at the student model; (2021-11-22)
99II.Related Work
